index.js

```diff
@@ -1 +1,682 @@
-console.log("hello world")
\ No newline at end of file
+ let toggle=document.querySelector('.toggle')
+let header=document.querySelector('header')
+let dropdown=document.querySelector('.drop-down ')
+let socials=document.querySelector('.socials-ul')
+let secondRow=document.querySelector('.second-roww ');
+let secondRoww=document.querySelector('.second-row ');
+ toggle.addEventListener('click',translate)
+ function translate(){
```

Comment:
No clue what you mean here. Remember to make your functions descriptive so you aren't the only one who knows what they are intended to do.

Reply:
i just added a comment for that